language/move-compiler/src/command_line/compiler.rs

```diff
@@ -465,7 +465,7 @@ pub fn construct_pre_compiled_lib<Paths: Into<Symbol>, NamedAddress: Into<Symbol
             typing = Some(tprog.clone())
         }
         PassResult::HLIR(hprog) => {
-            if env.has_diags() {
+            if env.has_diags_at_or_above_severity(Severity::NonblockingError) {
```

Comment:
The usage of `NoblockingError` here, but `BlockingError` above is puzzling to me.